### ISSUE TITLE
Should `vue/component-name-in-template-casing` be classified as `Essential`

### ISSUE THREAD

**Issue by xxholly32** [vuejs/eslint-plugin-vue#2474]
**What rule do you want to change?**

Should `vue/component-name-in-template-casing` be classified as `Essential`

**Does this change cause the rule to produce more or fewer warnings?**

more

**How will the change be implemented? (New option, new default behavior, etc.)?**

Adjustment classification

**Please provide some example code that this change will affect:**

```vue
/* ✗ BAD */
<el-button>Add ToDo</el-button>
/* ✓ GOOD */
<ElButton>Add ToDo</ElButton>
```

**What does the rule currently do for this code?**

https://vuejs.org/style-guide/rules-essential.html#use-multi-word-component-names

**What will the rule do after it's changed?**

**Additional context**

I think a lot of the code will be tweaked because so many components are named using multi-word. But this is an official recommendation why not categorize it in Essential Or am I wrong about the rule

**Comment by ota-meshi** [comment]
I personally agree with you, but `vue/component-name-in-template-casing` has been changed to uncategorized in the past.
#703
I would like to respect that.
Thank you.
